compose/docker-compose.ee.yml

```diff
@@ -67,21 +67,17 @@ services:
       - dnsconfig:/root/dnsconfig
   mq:
     container_name: mq
-    image: eclipse-mosquitto:2.0.15-openssl
-    depends_on:
-      - netmaker
+    image: emqx/emqx:5.0.17
```

Comment:
ee config should default to mq, there is a separate compose file for emqx

Reply:
but @0xdcarns said ee should default to emqx

Reply:
that is correct